Header header
string title
string database
string collection
string search_type
uint32 page_limit

string BROWSE_MOVIES=browse-movies
string BROWSE_SHOWS=browse-shows
string FIND_MOVIE=find-title